reports/builder.go

```diff
@@ -47,4 +54,5 @@ type Report struct {
 	SalVersion string
 	RunUUID    string
 	UserName   string
+	base64bz2report Base64bz2Report
```

Comment:
The rest of the fields are public(uppercase), but this one is private. What is the reasoning behind that?

Reply:
i think you caught me committing lazy code here.  the new code has this being exported.